packages/core-components/src/components/LogViewer/LogLine.test.tsx

```diff
@@ -354,3 +357,30 @@ describe('calculateHighlightedChunks', () => {
     ]);
   });
 });
+
+describe('linkify', () => {
+  it('should add links around URLs', async () => {
+    const classes = {};
+    const testText = new AnsiLine(1, [
+      {
+        text: 'Fetching template content from remote URL: https://github.com/backstage/software-templates/tree/main/scaffolder-templates/react-ssr-template/skeleton',
+        modifiers: {},
+      },
+    ]);
+
+    const rendered = await renderInTestApp(
+      <LogLine
+        line={testText}
+        classes={classes as Parameters<typeof getModifierClasses>[0]}
```

Comment:
This gave me some troubles so I just "stole" from the `getModifierClasses` test. Happy to change this with a better solution.